lib/rspec/collection_matchers/have.rb

```diff
@@ -24,6 +24,12 @@ def relativities
         }
       end
 
+      if RUBY_VERSION == '1.9.2'
+        def to_ary
+          [self]
+        end
+      end
```

Comment:
I don't see how we can, it's a broken behaviour on `1.9.2`, I'm ok with this for a subsidary gem, if it happens elsewhere we could revisit but it doesn't seem to.